### ISSUE TITLE
processing-video doesn't detect GStreamer files in sketch's code folder

### ISSUE THREAD

**Issue by luiscastilho** [processing/processing-video#204]
Hello,

It used to be possible to copy the contents of processing-video library (`Documents/Processing/libraries/video/library/*`) to the `code` folder of a sketch and Processing would detect it there and be able to play videos, for example. Even if `Documents/Processing/libraries` was empty, no libraries installed there.

That doesn't work with the latest release of processing-video (release 9 or 2.1). Trying to run the Loop example this way, Processing prints this error messages:
```
We could not find a system-wide or bundled installation of GStreamer, but video might still work if GStreamer was placed somewhere else
UnsatisfiedLinkError: Could not load library: gstreamer
UnsatisfiedLinkError: Could not load library: gstreamer
UnsatisfiedLinkError: Could not load library: gstreamer
A library used by this sketch relies on native code that is not available.
UnsatisfiedLinkError: Could not load library: gstreamer
UnsatisfiedLinkError: Could not load library: gstreamer
UnsatisfiedLinkError: Could not load library: gstreamer
```

Not sure if it's an issue or if it's intended behavior, but it's a nice feature to be able to bundle all libraries needed by a sketch in the `code` folder. I use it as a way to "freeze" versions that work well with a sketch. Is it possible for processing-video to check if there's a `code` folder in the sketch folder and if needed GStreamer files are there?

For now I'm using the code below to get processing-video to locate Gstreamer inside `code`. I couldn't test this in Linux or macOS but it works well on Windows.

```Java
if ( PApplet.platform == WINDOWS )
  System.setProperty("gstreamer.library.path", sketchPath() + "/code/windows-amd64");
else if ( PApplet.platform == MACOSX )
  System.setProperty("gstreamer.library.path", sketchPath() + "/code/macos-x86_64");
else if ( PApplet.platform == LINUX )
  System.setProperty("gstreamer.library.path", sketchPath() + "/code/linux-amd64");
System.setProperty("gstreamer.plugin.path", System.getProperty("gstreamer.library.path") + "/gstreamer-1.0");
```

Environment:
* Processing version: 4.0.1
* Operating System and OS version: Windows 10 Pro v21H2
* Other information: processing-video release 9 (2.1) - `library/*` contents copied to sketch's `code` folder and library removed from `Documents/Processing/libraries`.

**Comment by luiscastilho** [comment]
The workaround above doesn't work with release 10 (2.2). Works fine with release 9 (2.1), at least in Windows.

**Comment by codeanticode** [comment]
Do you have a other version of GStreamer installed on your computer?

**Comment by luiscastilho** [comment]
No, I don't. In fact I have never installed GStreamer on my machine, so no chance of having left overs of a previous installation or anything like that.

And as I said the workaround I found was working with version 9/2.1 but stopped working with version 10/2.2. Not sure what changed.

**Comment by mike2moon** [comment]
spent some time looking for a solution for this and user here said to set the system property manually to your gstream install, didnt say how, so i asked chatgtp and this line of code worked for me 
System.setProperty("gstreamer.path", "C:\\gstreamer\\1.0\\x86_64");
You will need to replace the path "C:\\gstreamer\\1.0\\x86_64" with the actual path to your GStreamer installation.  
Mine was "C:\\gstreamer\\1.0\\msvc_x86_64" for example, the default after installing gstreamer from their website
